README.md

```diff
@@ -84,6 +84,20 @@ includes/libraries/etc. The `--sysroot=...` option is not necessary if
 `WASI_SDK_PATH` is `/opt/wasi-sdk`. For troubleshooting, one can replace the
 `--sysroot` path with a manual build of [wasi-libc].
 
+### Integrating with a CMake build system
+
+Use a toolchain file to setup the *wasi-sdk* platform.
+
+```
+$ cmake -DCMAKE_TOOLCHAIN_FILE=${wasi_sdk_home}/share/cmake/wasi-sdk.cmake -S . -B build
```

Comment:
How about just `cmake -DCMAKE_TOOLCHAIN_FILE=${wasi_sdk_home}/share/cmake/wasi-sdk.cmake ...`  .. All those other arguments are very project-specific.

Reply:
Also, can you use `WASI_SDK_PATH` rather than `wasi_sdk_home` for consistency with the existing example code above?